src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/ReInviteUserFuncTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.userprofileapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.userprofileapi.controller.advice.ErrorResponse;
+import uk.gov.hmcts.reform.userprofileapi.controller.response.UserProfileCreationResponse;
+import uk.gov.hmcts.reform.userprofileapi.resource.UserProfileCreationData;
+
+@RunWith(SpringIntegrationSerenityRunner.class)
+public class ReInviteUserFuncTest extends AbstractFunctional {
```

Comment:
1. For happy path Integration test cases are covered. Functional test cases can't be implemented for  happy path since it requires tests to wait for resendInterval.
2. All Scenarios are covered in Integration test cases. Also for reinviteuser = false , existing Functional and Integration will take care.